src/Microsoft.Azure.SignalR.Management/Serialization/JsonObjectSerializerHubProtocol.cs

```diff
@@ -57,15 +57,15 @@ internal sealed class JsonObjectSerializerHubProtocol : IHubProtocol
         private const string ProtocolName = "json";
         private const int ProtocolVersion = 1;
 
-        private readonly ObjectSerializer _objectSerializer;
```

Comment:
It has to be public later when implementing full support for MessagePack in transient mode. Make it public first for tests.